.github/workflows/main.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-latest, ubuntu-latest]
+        os: [macos-latest, ubuntu-latest, windows-latest]
         dafny: [2.3.0, 3.0.0, 3.1.0, 3.2.0, 3.3.0, 3.4.0]
```

Comment:
While we're touching this could you update this list? We should at least test the latest, and "nightly" now that you've added that option.

Reply:
added, not sure how to add a nightly string

Reply:
I think you just add the string `"nightly"` to the `dafny:` array.

Reply:
@tchajed The `nightly` tag has many assets.
To specify one you need to qualify what "version" it is.
e.g.
https://github.com/dafny-lang/dafny/releases/download/nightly/dafny-nightly-2022-09-29-13352e7-x64-ubuntu-16.04.zip

So in this case the `dafny-version` would be `nightly-2022-09-29-13352e7`.
However, when I added that to the array,
the GHA did not run. :(